frontend/src/components/AddingPlayerForm/AddingPlayerForm.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { Button, TextField, Typography } from "@material-ui/core";
+import { Centered } from "../Shared/Centered";
+import { VerticalSpace } from "../Shared/VerticalSpace";
+
+function AddingPlayerForm(props: { title: string; inputs: string[] }) {
```

Comment:
oj, oj, oj. Po co to title i inputs jako propsy? 
Props to coś w stylu. Chcesz wyswietlac dla innego playera jego profil to podajesz playerIf w propsach. Tutaj to raczej nam tylko przeszkadza. Skoro to jest AddingPlayerForm, a nie żaden inny to inputy i title będą takie same. A podając Stringa sobie ograniczasz możliwości np. zrobienia specjalnego inputa na email / telefon itp.

Reply:
I pomysł też o testach. Dlatego dobrze zaczynać od testów :) Np. Będzie test, że formularz dodawania gracza ma następujące pola... A co to za test skoro te pola musisz podać z zewnątrz :)?

Reply:
haha najpierw tak zrobiłem i potem stwierdziłem, że zrobię coś bardziej ogólnego do użycia później. Też się zastanawiałem nad polami typu passowrd itp - tutaj o tyle dobrze, że są stringi. 
Zmieniam oczywiście na wersję wg tego co napisałeś :-)